application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsDashboardClass.xml

```diff
@@ -0,0 +1,218 @@
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <version>1.1</version>
+  <title>AnalyticsDashboard Class</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>true</hidden>
+  <content>{{velocity}}
+## Replace the default space with the space where you want your documents to be created.
+## Replace the default parent with the one of your choice and save the document.
+##
+#set ($defaultSpace = 'Analytics')
+#set ($defaultParent = 'Analytics.WebHome')
+{{/velocity}}</content>
+  <class>
+    <name>Analytics.Code.AnalyticsDashboardClass</name>
+    <customClass/>
+    <customMapping/>
+    <defaultViewSheet/>
+    <defaultEditSheet/>
+    <defaultWeb/>
+    <nameField/>
+    <validationScript/>
+    <description>
+      <contenttype>---</contenttype>
+      <customDisplay/>
+      <disabled>0</disabled>
+      <editor>---</editor>
+      <hint>Description of the dashbord.</hint>
+      <name>description</name>
+      <number>3</number>
+      <picker>1</picker>
+      <prettyName>Description</prettyName>
+      <rows>5</rows>
+      <size>40</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+    </description>
+    <displayedStatistics>
+      <cache>0</cache>
+      <customDisplay/>
+      <defaultValue/>
+      <disabled>0</disabled>
+      <displayType>radio</displayType>
+      <freeText/>
+      <hint>You should selected the widgets that you want to display.</hint>
+      <largeStorage>0</largeStorage>
+      <multiSelect>1</multiSelect>
+      <name>displayedStatistics</name>
+      <number>4</number>
+      <picker>1</picker>
+      <prettyName>Displayed Statistics</prettyName>
+      <relationalStorage>0</relationalStorage>
+      <separator> </separator>
+      <separators> ,|</separators>
+      <size>1</size>
+      <sort/>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <values>Random | Words | For | Now</values>
+      <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+    </displayedStatistics>
+    <endDate>
+      <customDisplay/>
+      <dateFormat>dd/MM/yyyy</dateFormat>
+      <disabled>0</disabled>
+      <emptyIsToday>0</emptyIsToday>
+      <hint>Enter the date that marks the end point for calculating statistics.</hint>
+      <name>endDate</name>
+      <number>2</number>
+      <picker>1</picker>
+      <prettyName>End Date</prettyName>
+      <size>20</size>
+      <unmodifiable>0</unmodifiable>
+      <validationMessage/>
+      <validationRegExp/>
+      <classType>com.xpn.xwiki.objects.classes.DateClass</classType>
+    </endDate>
+    <startDate>
+      <customDisplay/>
+      <dateFormat>dd/MM/yyyy</dateFormat>
+      <disabled>0</disabled>
+      <emptyIsToday>0</emptyIsToday>
+      <hint>Specify the starting date from which you want the statistics to be calculated.</hint>
```

Comment:
Also, I think this hint can be improved. Maybe `Start time of the date interval used on all statistics of this dashboard.` or `Start time for the statistics date interval. This will be used for all statistics of this dashboard.` 
Same for endDate